examples/python/arkitscenes/main.py

```diff
@@ -0,0 +1,198 @@
+        tuple: A tuple containing the timestamp (ts) and the transformation matrix (Rt).
+
+    Raises
+    ------
+        AssertionError: If the input string does not contain 7 tokens.
+    """
+    tokens = traj_string.split()  # Split the input string into tokens
+    assert len(tokens) == 7, f"Input string must have 7 tokens, but found {len(tokens)}."
+    ts: str = tokens[0]  # Extract timestamp from the first token
+
+    # Extract rotation from the second to fourth tokens
+    angle_axis = [float(tokens[1]), float(tokens[2]), float(tokens[3])]
+    r_w_to_p = cv2.Rodrigues(np.asarray(angle_axis))[0]
+
+    # Extract translation from the fifth to seventh tokens
+    t_w_to_p = np.asarray([float(tokens[4]), float(tokens[5]), float(tokens[6])])
+
+    # Construct the extrinsics matrix
+    extrinsics = np.eye(4, 4)
+    extrinsics[:3, :3] = r_w_to_p
+    extrinsics[:3, -1] = t_w_to_p
+
+    # Compute the inverse of extrinsics matrix to get the rotation and translation matrices
+    Rt = np.linalg.inv(extrinsics)
+
+    return (ts, Rt)
+
+
+def log_arkit(recording_path: Path) -> None:
+    """
+    Logs ARKit recording data using Rerun.
+
+    Args:
+        recording_path (Path): The path to the ARKit recording.
+
+    Returns
+    -------
+        None
+    """
+    video_id = recording_path.stem
+    image_dir = recording_path / "lowres_wide"
+    depth_dir = recording_path / "lowres_depth"
+    intrinsics_dir = recording_path / "lowres_wide_intrinsics"
+    traj_path = recording_path / "lowres_wide.traj"
+
+    frame_ids = [x.name for x in sorted(depth_dir.iterdir())]
+    frame_ids = [x.split(".png")[0].split("_")[1] for x in frame_ids]
+    frame_ids.sort()
+
+    poses_from_traj = {}
+    with open(traj_path, "r", encoding="utf-8") as f:
+        traj = f.readlines()
+
+    for line in traj:
+        ts, Rt = traj_string_to_matrix(line)
+        # round timestamp to 3 decimal places
+        ts = f"{round(float(ts), 3):.3f}"
+        poses_from_traj[ts] = Rt
+
+    ply_path = recording_path / f"{recording_path.stem}_3dod_mesh.ply"
+    bbox_annotations_path = recording_path / f"{recording_path.stem}_3dod_annotation.json"
+    annotation = load_json(bbox_annotations_path)
+    log_annotated_bboxes(annotation)
+
+    # TODO(pablovela5620): Wait for resolution of either #1570 or #1571 for textured mesh
+    # for now just use the untextered/uncolored mesh
+    # # convert ply to obj
+    # obj_path = recording_path / f"{recording_path.stem}_3dod_mesh.obj"
+    # if not obj_path.exists():
+    #     mesh = o3d.io.read_triangle_mesh(str(ply_path))
+    #     o3d.io.write_triangle_mesh(str(obj_path), mesh)
+
+    ply = trimesh.load(str(ply_path))
+    verts = ply.vertices
+    faces = ply.faces
+
+    # Log the mesh
+    rr.log_mesh("world/mesh_log", positions=verts, indices=faces, timeless=True)
+
+    # with open(obj_path, mode="rb") as file:
+    #     rr.log_mesh_file("world/mesh_log_file", MeshFormat.OBJ, file.read(), timeless=True)
```

Comment:
```suggestion
    mesh_ply = trimesh.load(str(ply_path))
    mesh_gltf_bin = trimesh.exchange.gltf.export_glb(ply, include_normals=True)
    rr.log_mesh_file("world/mesh", MeshFormat.GLB, mesh_gltf_bin)
```

(Note: you also need to `from rerun.log.file import MeshFormat`)

You could also log it with `log_mesh` but this way you'll automatically get vertex colors without changing the code when #1580 lands.